src/components/v2/V2Create/forms/TokenForm/index.tsx

```diff
@@ -252,14 +273,60 @@ export default function TokenForm({
     formatDiscountRate(BigNumber.from(discountRate)),
   )
 
+  // Total tokens minted as a result of a 1 ETH contribution
+  const initialMintingRate = parseFloat(weight) ?? DEFAULT_ISSUANCE_RATE
+  const reservedRateDecimal = reservedRatePercent * 0.01
   // Tokens received by contributor's per ETH
   const initialIssuanceRate =
-    DEFAULT_ISSUANCE_RATE - reservedRatePercent * MAX_RESERVED_RATE
+    initialMintingRate - reservedRateDecimal * initialMintingRate
 
   return (
     <Form layout="vertical" onFinish={onTokenFormSaved}>
       <Space size="middle" direction="vertical">
         <div>
+          <div style={{ display: 'flex' }}>
+            {isCreate ? (
+              <Form.Item
+                label={
+                  <TooltipLabel
+                    label={<Trans>Initial inflation rate</Trans>}
```

Comment:
**suggestion**: I don't think inflation rate is the best term here.

I am leaning towards `Initial mint rate`

```suggestion
                    label={<Trans>Initial mint rate</Trans>}
```